docker/Dockerfile.gpu

```diff
@@ -16,17 +16,17 @@ RUN apt-get update && \
     && curl -O https://bootstrap.pypa.io/get-pip.py \
     && python get-pip.py
 
-RUN pip install mxnet-model-server \
-    && pip install mxnet-cu90mkl
+RUN pip install --no-cache-dir mxnet-model-server \
+    && pip install --no-cache-dir mxnet-cu92mkl==1.3.0.post0
```

Comment:
Same for MXNet version here.

Reply:
Same as above ... Once the MMS is released, I would add that version to this file as well. People who want the latest can work with the nightly builds, but for release versions, we should keep the container environment as constant as possible.

Reply:
If we pass in version, we can use single docker file. Which will be easier to maintain. And we have a docker-release.sh script which hard code all the versions.